### PR TITLE
Polls show

### PR DESCRIPTION
**Pull request by decabeza**
Where
=====
* **Related Issue:** Polls: design #1955

What
====
- New design for `polls/show` view.
- Include image poll on `polls/index` view.

Warnings ⚠️
========
- This view has some **lorem and example content**, also some comments to reference to new backend functionalities we are working on.

Screenshots
===========
polls_index

polls_show

### CODE REVIEW

File: app/views/polls/_gallery.html.erb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<div class="orbit margin-bottom" role="region" aria-label="Answer 1" data-orbit data-auto-play="false">
+  <a data-toggle="answer_1" class="zoom-link show-for-medium-up">
+    <span class="icon-search-plus"></span>
+    <span class="show-for-sr"><%= t("polls.show.zoom_plus") %></span>
+  </a>
+
+  <ul class="orbit-container">
+    <li>
+      <button class="orbit-previous">
+        <span class="show-for-sr">Previous Slide</span>&#9664;&#xFE0E;
```
Comment (bertocq) [inline app/views/polls/_gallery.html.erb]: Translation needed

File: app/views/polls/_gallery.html.erb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      </button>
+    </li>
+    <li>
+      <button class="orbit-next">
+        <span class="show-for-sr">Next Slide</span>&#9654;&#xFE0E;
```
Comment (bertocq) [inline app/views/polls/_gallery.html.erb]: Translation needed

File: app/views/polls/_gallery.html.erb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        <%= image_tag "example_horizontal.jpg", class: "orbit-image" %>
+      <% end %>
+      <span class="orbit-caption">Image title 1</span>
+    </li>
+    <!-- end -->
+    <li class="orbit-slide">
+      <%= link_to "/assets/example_vertical.jpg", target: "_blank" do %>
+        <%= image_tag "example_vertical.jpg", class: "orbit-image" %>
+      <% end %>
+      <span class="orbit-caption">Image title 2</span>
```
Comment (bertocq) [inline app/views/polls/_gallery.html.erb]: I suppose this is just place holder and we'll do `image.title` but a comment to remember will help :) same  on bellow lines 36 and 39 👍

Comment (decabeza) [reply app/views/polls/_gallery.html.erb]: Changes done!

**Pull request merged**